docs/modules/secret-operator/pages/secretclass.adoc

```diff
@@ -134,8 +134,7 @@ spec:
     Reference (`name` and `namespace`) to a K8s `Secret` object where the trusted certificates are stored.
     The extension of a key defines its content:
     * `.der` denotes a a binary DER certificates.
-    * `.pem` denotes a stack of base64 encoded DER certificates.
-    * `.cer`, `.cert` and `.crt` denote either a binary DER certificate or a stack of base64 encoded DER certificates.
+    * `.crt` denotes a stack of base64 encoded DER certificates.
```

Comment:
```suggestion
    * `.crt` denotes a stack of PEM (base64-encoded DER) certificates.
```

Reply:
I'd also reorder these to put .crt first.

Reply:
done (834f3b2a0a265baf1e9e49b2e43d9379d08b2142)